src/HubspotProvider.tsx

```diff
@@ -17,22 +19,53 @@ interface HubspotProviderProps {
   readonly async?: boolean;
   readonly addToHead?: boolean;
   readonly children: React.ReactNode;
+  readonly hubspotDestinationUrl?: string;
+  readonly region?: string;
 }
 
 const HubspotProvider = ({
   async,
   addToHead,
-  children
+  children,
+  hubspotDestinationUrl,
+  region
 }: HubspotProviderProps): ReactElement => {
+  // Check if we're nested inside another HubspotProvider
+  const parentContext = useContext(HubspotContext);
```

Comment:
Is check this really necessary?  In what scenario is this context ever going to be nested?  

Adds a bunch of checks I don't think we need.